### PR TITLE
Enhance pagination classes

### LINKED ISSUE #5179: Same classes for pods pagination numbers links and labels links while using shortcode
## Issue Overview
The pagination HTML code rendered by the shortcode use displays the same CLASSES for : 
- The next/previous, first/last page NUMBERS and
- The "Next >", "< Previous", "First" and "Last" LABELS
It's difficult then to style those items separatly or to make the page numbers displayed without the next/previous/first/last labels....

## Expected Behavior
It would be great if the HTML rendering of the function called by the shortcode would give as least one different class to those items so we can style them individually.
Ex : 
Actual HTML rendering for the first page links :
"a href="/accueil/medias/?pg=1" class="pods-pagination-number pods-pagination-first ">« First</a		
[...]
a href="/accueil/medias/?pg=1" class="pods-pagination-number pods-pagination-first  ">1</a"

Versus the Ideal HTML rendering for the first page links :

"a href="/accueil/medias/?pg=1" class="pods-pagination-number pods-pagination-first PODS-PAGINATION-FIRST-LABEL ">« First</a 		
[...]
a href="/accueil/medias/?pg=1" class="pods-pagination-number pods-pagination-first PODS-PAGINATION-FIRST-NUMBER  ">1</a"

### PR DESCRIPTION
**Pull request by JoryHogeveen**
## Description

Use correct classes for pagination items.

## Related GitHub issue(s)

Fixes #5179 

## PR checklist

- [x] I have tested my own code to confirm it works as I intended.
- [x] My code follows the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- [x] My code follows the [WordPress Inline Documentation Standards](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/).

### CODE REVIEW

File: ui/front/pagination/advanced.php [hunk 1]
```diff
@@ -7,13 +7,13 @@
 if ( 1 < $params->page ) {
 	?>
 	<?php if ( $params->first_last ) { ?>
-			<a href="<?php echo esc_url( pods_query_arg( array( $params->page_var => 1 ) ) ); ?>" class="pods-pagination-number pods-pagination-first <?php echo esc_attr( $params->link_class ); ?>"><?php echo $params->first_text; ?></a>
+			<a href="<?php echo esc_url( pods_query_arg( array( $params->page_var => 1 ) ) ); ?>" class="pods-pagination-label pods-pagination-first <?php echo esc_attr( $params->link_class ); ?>"><?php echo $params->first_text; ?></a>
```
Comment (sc0ttkclark) [inline ui/front/pagination/advanced.php:10]: Backcompat classes

```suggestion
			<a href="<?php echo esc_url( pods_query_arg( array( $params->page_var => 1 ) ) ); ?>" class="pods-pagination-label pods-pagination-number pods-pagination-first <?php echo esc_attr( $params->link_class ); ?>"><?php echo $params->first_text; ?></a>
```

File: ui/front/pagination/advanced.php [hunk 2]
```diff
@@ -7,13 +7,13 @@
...
 		<?php } ?>
 	<?php if ( $params->prev_next ) { ?>
-			<a href="<?php echo esc_url( pods_query_arg( array( $params->page_var => ( $params->page - 1 ) ) ) ); ?>" class="pods-pagination-number pods-pagination-prev <?php echo esc_attr( $params->link_class ); ?>"><?php echo $params->prev_text; ?></a>
+			<a href="<?php echo esc_url( pods_query_arg( array( $params->page_var => ( $params->page - 1 ) ) ) ); ?>" class="pods-pagination-label pods-pagination-prev <?php echo esc_attr( $params->link_class ); ?>"><?php echo $params->prev_text; ?></a>
```
Comment (sc0ttkclark) [inline ui/front/pagination/advanced.php:13]: Backcompat classes

```suggestion
			<a href="<?php echo esc_url( pods_query_arg( array( $params->page_var => ( $params->page - 1 ) ) ) ); ?>" class="pods-pagination-label pods-pagination-number pods-pagination-prev <?php echo esc_attr( $params->link_class ); ?>"><?php echo $params->prev_text; ?></a>
```

File: ui/front/pagination/advanced.php [hunk 3]
```diff
@@ -63,14 +63,14 @@
 
 	if ( $params->page < $params->total ) {
 		?>
-		<a href="<?php echo esc_url( pods_query_arg( array( $params->page_var => $params->total ) ) ); ?>" class="pods-pagination-number pods-pagination-last <?php echo esc_attr( $params->link_class ); ?>"><?php echo $params->total; ?></a>
+		<a href="<?php echo esc_url( pods_query_arg( array( $params->page_var => $params->total ) ) ); ?>" class="pods-pagination-number pods-pagination-last pods-pagination-<?php echo esc_attr( $params->total ); ?> <?php echo esc_attr( $params->link_class ); ?>"><?php echo $params->total; ?></a>
 		<?php
 		if ( $params->prev_next ) {
 			?>
-			<a href="<?php echo esc_url( pods_query_arg( array( $params->page_var => ( $params->page + 1 ) ) ) ); ?>" class="pods-pagination-number pods-pagination-next <?php echo esc_attr( $params->link_class ); ?>"><?php echo $params->next_text; ?></a>
+			<a href="<?php echo esc_url( pods_query_arg( array( $params->page_var => ( $params->page + 1 ) ) ) ); ?>" class="pods-pagination-label pods-pagination-next <?php echo esc_attr( $params->link_class ); ?>"><?php echo $params->next_text; ?></a>
```
Comment (sc0ttkclark) [inline ui/front/pagination/advanced.php:70]: Backcompat classes

```suggestion
			<a href="<?php echo esc_url( pods_query_arg( array( $params->page_var => ( $params->page + 1 ) ) ) ); ?>" class="pods-pagination-label pods-pagination-number pods-pagination-next <?php echo esc_attr( $params->link_class ); ?>"><?php echo $params->next_text; ?></a>
```

File: ui/front/pagination/advanced.php [hunk 4]
```diff
@@ -63,14 +63,14 @@
...
 		<?php } ?>
 		<?php if ( $params->first_last ) { ?>
-			<a href="<?php echo esc_url( pods_query_arg( array( $params->page_var => $params->total ) ) ); ?>" class="pods-pagination-number pods-pagination-last <?php echo esc_attr( $params->link_class ); ?>"><?php echo $params->last_text; ?></a>
+			<a href="<?php echo esc_url( pods_query_arg( array( $params->page_var => $params->total ) ) ); ?>" class="pods-pagination-label pods-pagination-last <?php echo esc_attr( $params->link_class ); ?>"><?php echo $params->last_text; ?></a>
```
Comment (sc0ttkclark) [inline ui/front/pagination/advanced.php:73]: Backcompat classes

```suggestion
			<a href="<?php echo esc_url( pods_query_arg( array( $params->page_var => $params->total ) ) ); ?>" class="pods-pagination-label pods-pagination-number pods-pagination-last <?php echo esc_attr( $params->link_class ); ?>"><?php echo $params->last_text; ?></a>
```

**Review by sc0ttkclark — CHANGES_REQUESTED** [review]
Minor tweaks needed

File: ui/front/pagination/advanced.php (hunk 1 above)
Comment (JoryHogeveen) [reply ui/front/pagination/advanced.php:10]: The whole purpose of this PR is to remove such invalid class names.

File: ui/front/pagination/advanced.php (hunk 2 above)
Comment (JoryHogeveen) [reply ui/front/pagination/advanced.php:13]: The whole purpose of this PR is to remove such invalid class names.

File: ui/front/pagination/advanced.php (hunk 3 above)
Comment (JoryHogeveen) [reply ui/front/pagination/advanced.php:70]: The whole purpose of this PR is to remove such invalid class names.

File: ui/front/pagination/advanced.php (hunk 4 above)
Comment (JoryHogeveen) [reply ui/front/pagination/advanced.php:73]: The whole purpose of this PR is to remove such invalid class names.

**Review by sc0ttkclark — APPROVED** [review]

**Review by sc0ttkclark — APPROVED** [review]

**Pull request merged**